taskipy/pyproject.py

```diff
@@ -27,6 +27,10 @@ def tasks(self) -> Dict[str, Task]:
         except KeyError:
             raise MissingTaskipyTasksSectionError()
 
+    @property
+    def variables(self) -> Dict[str, Task]:
+        return self.__items['tool']['taskipy'].get('variables', {})
```

Comment:
All of the nested keys might be missing (`tool`, `taskipy`, and `variables`). Therefore, perhaps we should consider wrapping the entire line with `try except`?

```suggestion
        try:
            return self.__items['tool']['taskipy'].get('variables', {})
        except KeyError:
            return {}
```